src/witan/send/vis/projection_charts.clj

```diff
@@ -0,0 +1,97 @@
+                                      ;; only a subset so the charts line up
+                                      (filter #(<= 2015 (:calendar-year %)))
+                                      (filter #(< (:academic-year %) 21)))
+                                     (vis.gp/general-population (str (:project-dir config) "/" (get-in config [:file-inputs :population]))))
+        census                 (vis.transitions/->census historical-transitions)
+        settings-lookup        (into {}
+                                     (map (fn [x] [(:setting x) (:setting x)]))
+                                     census)
+        needs-lookup           (into {}
+                                     (map (fn [x] [(:setting x) (:setting x)]))
+                                     census)
+        output-ay              (future
+                                 (vis.oay/charts
+                                  (vis.transitions/ay-counts-per-calendar-year historical-transitions)
+                                  (into []
+                                        (filter #(< (:academic-year %) 21))
+                                        (vis.oay/output-ay (str output-dir "/" vis.oay/output-ay-file)))))
+        output-count           (future
+                                 (vis.ocount/chart
+                                  "2020 Baseline"
+                                  (vis.transitions/total-population-per-calendar-year historical-transitions)
+                                  (vis.ocount/output-count (str output-dir "/" vis.ocount/output-count-file))))
+        output-cost            (future
+                                 (vis.ocost/chart
+                                  "2020 Baseline"
+                                  (vis.ocost/output-cost (str output-dir "/" vis.ocost/output-cost-file))))
+        output-need            (future
+                                 (vis.on/charts
+                                  needs-lookup
+                                  (vis.transitions/needs-counts-per-calendar-year historical-transitions)
+                                  (vis.on/output-need (str output-dir "/" vis.on/output-need-file))
+                                  need-titles-and-sets))
+        output-setting         (future
+                                 (vis.os/charts
+                                  settings-lookup
+                                  (vis.transitions/settings-counts-per-calendar-year historical-transitions)
+                                  (vis.os/output-setting (str output-dir "/" vis.os/output-setting-file))
+                                  setting-titles-and-sets))
+        output-setting-cost    (vis.osc/charts
+                                (vis.osc/output-setting-cost (str output-dir "/" vis.osc/output-setting-cost-file))
+                                setting-titles-and-sets)
+        general-population     (future (vis.gp/charts gen-pop-data))]
+
+    (a/thread
```

Comment:
So this is using `thread` because it is dependent on `output-ay` but you still want it to run on a new, unsued thread correct? And I guess all of this parallelisation (is that even what it actually is?) speed is only achieved based on how many cores you provide to the process. Does Clojure just make use of how ever many you provide it. I know some other languages you still have to make it much clearer how you want the resources to be used.